css/elements.css

```diff
@@ -1285,12 +1326,16 @@ li.menu-search-result-term:before {
   flex-direction: column;
 }
 
+#references-pane > .menu-pane-header {
+  cursor: row-resize;
+}
+
 #references-pane-container.active {
   display: flex;
 }
 
 #references-pane-close:after {
-  content: '✖';
+  content: '\2716';
```

Comment:
This is a drive-by fix for an encoding error I got when doing local testing from a `file://` URL.

Reply:
... is that seriously how one spells hex escapes in CSS? Gross.

Reply:
yep https://drafts.csswg.org/css-syntax/#escaping